service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerSubmittedTest.java

```diff
@@ -58,44 +69,107 @@
 import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
 
 class ManageHearingsControllerSubmittedTest extends ManageHearingsControllerTest {
 
     private static final long CASE_ID = 12345L;
+    private static final String FAMILY_MAN_NUMBER = "FMN1";
     private static final long ASYNC_METHOD_CALL_TIMEOUT = 10000;
     private static final String JUDGE_EMAIL = "judge@judge.com";
     private static final String CAFCASS_EMAIL = "cafcass@cafcass.com";
     private static final String NOTIFICATION_REFERENCE = "localhost/" + CASE_ID;
 
+    private static final UUID LETTER_1_ID = randomUUID();
+    private static final UUID LETTER_2_ID = randomUUID();
+    private static final Document NOTICE_OF_HEARING_DOCUMENT = testDocument();
+    private static final Document COVERSHEET_REPRESENTATIVE = testDocument();
+    private static final Document COVERSHEET_RESPONDENT = testDocument();
+    private static final byte[] NOTICE_OF_HEARING_BINARY = testDocumentBinary();
+    private static final byte[] COVERSHEET_REPRESENTATIVE_BINARY = testDocumentBinary();
+    private static final byte[] COVERSHEET_RESPONDENT_BINARY = testDocumentBinary();
+
     private final Element<HearingBooking> hearingWithoutNotice = element(HearingBooking.builder()
         .type(CASE_MANAGEMENT)
         .startDate(LocalDateTime.of(2050, 5, 20, 13, 0))
         .endDate(LocalDateTime.of(2050, 5, 20, 14, 0))
         .noticeOfHearing(null)
         .build());
 
+    private static final Element<Representative> REPRESENTATIVE_POST = element(Representative.builder()
+        .fullName("First Representative")
+        .servingPreferences(POST)
+        .address(testAddress())
+        .build());
+
+    private static final Element<Representative> REPRESENTATIVE_EMAIL = element(Representative.builder()
+        .fullName("Third Representative")
+        .servingPreferences(EMAIL)
+        .email("third@representatives.com")
+        .build());
+
+    private static final Element<Representative> REPRESENTATIVE_DIGITAL = element(Representative.builder()
+        .fullName("Second Representative")
+        .servingPreferences(DIGITAL_SERVICE)
+        .email("second@representatives.com")
+        .build());
+
+    private static final Respondent RESPONDENT_NOT_REPRESENTED = Respondent.builder()
+        .party(RespondentParty.builder()
+            .firstName("Alex")
+            .lastName("Jones")
+            .address(testAddress())
+            .build())
+        .build();
+
+    private static final Respondent RESPONDENT_REPRESENTED = Respondent.builder()
+        .party(RespondentParty.builder()
+            .firstName("George")
+            .lastName("Jones")
+            .address(testAddress())
+            .build())
+        .representedBy(wrapElements(REPRESENTATIVE_POST.getId(), REPRESENTATIVE_DIGITAL.getId()))
+        .build();
+
+
+    @Captor
+    private ArgumentCaptor<Map<String, Object>> caseDetails;
```

Comment:
Minor: Although the compiler takes care of scope and context of variable but a lot of older tests in this class use a local variable named caseDetails so it might be confusing to read if both a variables will be used in same function. Just my opinionated preference so no need to change if it is acceptable by wider audience.

Reply:
done